docs/content/preview/yugabyte-voyager/release-notes.md

```diff
@@ -13,6 +13,20 @@ type: docs
 
 What follows are the release notes for the YugabyteDB Voyager v1 release series. Content will be added as new notable features and changes are available in the patch releases of the YugabyteDB v1 series.
 
+## v1.8.3 - October 15, 2024
+
+### Enhancements
+
+- The analyze-schema now reports the [unsupported datatypes](../known-issues/postgresql/#unsupported-datatypes-by-yugabytedb) by YugabyteDB (such as PostGIS types, geometry native types, and so on) and the [datatypes not supported during live migration](../known-issues/postgresql/#unsupported-datatypes-by-voyager-during-live-migration).
+- The assessment report now reports the PostGIS datatypes (geometry and geography) under "Unsupported datatypes".
+- The assess-migration command now reports the unsupported BRIN and SPGIST index methods from PostgreSQL source.
+- Improved console output of the export schema command in case of MVIEW sizing recommendations.
+
+### Bug fixes
+
+- Fixed an issue in the streaming phase of the import data command, which occurred only in case of unique key conflicting events. The producer [goroutine](https://go.dev/tour/concurrency/1) (which checks for conflicts, calculates the event's hash, and places them on a channel for consumption) was blocked because the channel was full and the consumer goroutine after processing these events, couldn't clear the channel due to a lock held by the producer, causing a deadlock.
```

Comment:
```suggestion
- Fixed an issue in the streaming phase of the import data command that only occurred in the case of unique key conflict events. The producer [goroutine](https://go.dev/tour/concurrency/1) (which checks for conflicts, calculates the event's hash, and places them on a channel for consumption) was blocked because the channel was full and, after processing these events, the consumer goroutine couldn't clear the channel due to a lock held by the producer, causing a deadlock.
```